### ISSUE TITLE
Format string warning in SSL_SESSION_print (probably low priority)

### ISSUE THREAD

**Issue by crd477** [libressl-portable/portable#208]
Platform: CYGWIN_NT-10.0 zzz 2.5.1(0.297/5/3) 2016-04-21 22:14 x86_64 Cygwin
Compiler: x86_64-w64-mingw32-gcc (GCC) 4.9.2
LibreSSL version: 2.3.6

```
[...]
  CC       ssl_txt.lo
ssl_txt.c: In function 'SSL_SESSION_print':
ssl_txt.c:166:3: warning: unknown conversion type character 'l' in format [-Wformat=]
   if (BIO_printf(bp, "\n    Start Time: %lld", (long long)x->time) <= 0)
   ^
ssl_txt.c:166:3: warning: too many arguments for format [-Wformat-extra-args]
[...]
```

**Comment by bob-beck** [comment]
lld is a legitimate C format code.   looks like a compiler/library bug to me.  Brent can we verify this?

**Comment by busterb** [comment]
%lld is somewhat sketchily handled on different windows platforms, but the warning is probably justified. the correct formatter if we really mean '64-bit integer' on windows is %l64d.

I had suggested something slightly gross a couple of years ago, but we couldn't decide if we wanted to use inttypes.h formatter macros to make this portable. I guess I could include it as a local patch for -portable instead.

```
-               if (BIO_printf(bp, "\n    Start Time: %lld", (long long)x->time) <= 0)
+               if (BIO_printf(bp, "\n    Start Time: %"PRId64, (int64_t)x->time) <= 0)
```

**Comment by busterb** [comment]
Fixed in fa435db8df9fca0220a69343ffed0aa6d1172097
